GDJS/Runtime/indexeddb.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2016 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  export const loadFromIndexedDB = async function (
```

Comment:
This will directly "pollute" the gdjs namespace which is already huge: https://docs.gdevelop.io/GDJS%20Runtime%20Documentation/modules/gdjs.html

Instead, let's move these into a namespace `indexedDb` (camelCase - PascalCase is only for classes).